test/test_eos_tabulated.cpp

```diff
@@ -269,12 +269,16 @@ SCENARIO("SpinerEOS and EOSPAC Serialization",
     SpinerEOSDependsRhoT rhoT_orig = SpinerEOSDependsRhoT(eosName, steelID);
     SpinerEOSDependsRhoSie rhoSie_orig = SpinerEOSDependsRhoSie(eosName, steelID);
     EOS eospac_orig = EOSPAC(steelID);
+    // not actually used but we want to stress test that we can serialize
+    // and deserialize multiple EOSPAC objects
+    EOS eospac_air = EOSPAC(airID);
```

Comment:
This object appears to be used. I'm confused by this comment

Reply:
Fair enough. Comment changed.